docs/en/ingest-management/fleet/fleet-api-docs.asciidoc

```diff
@@ -5,6 +5,5 @@
 through the {fleet} UI are also available through the API. In fact, the UI
 consumes the API.
 
-Please see the {kib} repo
-https://github.com/elastic/kibana/blob/master/x-pack/plugins/fleet/common/openapi/README.md[{fleet} API Documentation file]
-for more details.
+Please refer to the
+https://github.com/elastic/kibana/blob/master/x-pack/plugins/fleet/common/openapi/README.md[{fleet} OpenAPI file] in the {kib} repo for more details.
```

Comment:
```suggestion
https://github.com/elastic/kibana/blob/{branch}/x-pack/plugins/fleet/common/openapi/README.md[{fleet} OpenAPI file] in the {kib} repo for more details.
```

Reply:
Sorry there is one last thing I forgot to fix. We need to use a variable here, or the URL will always point to master and not necessarily reflect the correct version.